server/templates/pages/src/terms.html

```diff
@@ -43,5 +39,10 @@ <h1 id="fxa-tos-header">{{#t}}Terms of Service{{/t}}</h1>
         </div>
 
         <a id="about-mozilla" rel="author" target="_blank" href="https://www.mozilla.org/about/?utm_source=firefox-accounts&amp;utm_medium=Referral"></a>
+
+        <!--[if lt IE 10]>
+            <p class="browsehappy">{{#t}}You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.{{/t}}</p>
+        <![endif]-->
```

Comment:
Ignore this, I filed as a separate issue; #2257
Travis is green, somebody merge this bad boy!